celestia/Cargo.toml

```diff
@@ -4,11 +4,15 @@ version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
 
+#[lib]
+#crate-type = ["cdylib"]
+
```

Comment:
Are we planning on changing that for example for wasm? Or is it just leftover from editing?

Reply:
fixed